crates/utils/src/claims.rs

```diff
@@ -23,10 +26,11 @@ impl Claims {
     )
   }
 
-  pub fn jwt(local_user_id: i32, hostname: String) -> Result<Jwt, jsonwebtoken::errors::Error> {
+  pub fn jwt(local_user_id: i32) -> Result<Jwt, jsonwebtoken::errors::Error> {
     let my_claims = Claims {
-      local_user_id,
-      iss: hostname,
+      sub: local_user_id,
+      iss: Settings::get().hostname(),
+      iat: chrono::Utc::now().timestamp_millis() / 1000,
```

Comment:
Same here, use `timestamp()`.

Reply:
Also, I would prefer to import all the types, instead of doing `chrono::Utc` etc.